data/helpers.d/network

```diff
@@ -0,0 +1,22 @@
+# Normalize the url path syntax
+# Handle the slash at the beginning of path and its absence at ending
+# Return a normalized url path
+#
+# example: url_path=$(ynh_normalize_url_path $url_path)
+#          ynh_normalize_url_path example -> /example
+#          ynh_normalize_url_path /example -> /example
+#          ynh_normalize_url_path /example/ -> /example
+#
+# usage: ynh_normalize_url_path path_to_normalize
+# | arg: url_path_to_normalize - URL path to normalize before using it
+ynh_normalize_url_path () {
+	path=$1
+	test -n "$path" || ynh_die "ynh_normalize_url_path expect a URL path as first argument and received nothing."
+	if [ "${path:0:1}" != "/" ]; then    # If the first character is not a /
```

Comment:
Imho having the comments on the previous lines rather than at the end would help readability, but maybe that's just personnal taste